honeycombio/honeycomb-opentelemetry-web
# feat: ✨ Add browser smoke tests

## Which problem is this PR solving?
Adds basic browser smoke tests that test whether the OTel API is initialized and that traces are produced. This is not running in CI yet.

## Short description of the changes
- Add Cypress as a dependency
- Configure Cypress tests to check console output for whether API is initialized and traces are produced
- Added Docker config to run smoke tests with one command

## How to verify that this has the expected result
- Check out this branch
- Make sure you have docker installed
- Run `npm run test:smoke`
